FRAMEWORKNAME/Package.swift

```diff
@@ -10,6 +11,19 @@ import PackageDescription
 
 let package = Package(
     name: "FRAMEWORKNAME",
+    products: [
+        .library(
+            name: "FRAMEWORKNAME",
+            targets: ["FRAMEWORKNAME"]),
+        ],
     dependencies: [],
-    exclude: ["Tests"]
+    targets: [
+        .target(
+            name: "FRAMEWORKNAME",
+            dependencies: [],
+            path: "Sources"),
+        .testTarget(
+            name: "FRAMEWORKNAMETests",
+            dependencies: ["FRAMEWORKNAME"])
+    ]
```

Comment:
If we add path here, then we can avoid one level nesting inside FRAMEWORKNAMETests like below:

```
.testTarget(
     name: "FRAMEWORKNAMETests",
     dependencies: ["FRAMEWORKNAME"],
     path: "Tests"),
```

Reply:
:tada: that was an awesome suggestion, thank you so much! fixed now!